src/Microsoft.Health.Fhir.Api/Features/Routing/RouteNames.cs

```diff
@@ -39,6 +39,8 @@ internal static class RouteNames
 
         internal const string PostBundle = "PostBundle";
 
+        internal const string EverythingOperationById = "EverythingOperationById";
```

Comment:
>EverythingOperationById [](http://example.com/codeflow?start=30&length=23)

it doesn't do much, other than provide name for attribute.
Since this is operation, we need to provide information about that operation as well.

See `ResolveOperationDefinitionUrl` in `UrlResolver` and 
`src\Microsoft.Health.Fhir.Core/Data/OperationDefinition` folder with json files.